### PR TITLE
Make search bar in dashboard have a more clear message.

### PR DESCRIPTION
**Pull request by ze**
The current search bar in the users dashboard isn't very clear on what its functionality exactly is. The user can assume that it will search inside their projects or some other functionality.

This pull request allows for the search bar to have a more clear functionality.

Update: I should have provided an image to accompany this PR.
screenshot from 2018-03-25 02-56-54

### CODE REVIEW

**Comment by stsewd** [comment]
This is the search on the project scope

screenshot-2018-3-25 search read the docs

So, shouldn't it say Search in Read the Docs? (I'm not sure if that it's right wording)

**Comment by ze** [comment]
My rationale was that since it has the same functionality as the search box at readthedocs.org 
screenshot from 2018-03-25 02-38-32 it would be appropriate if it had the same text.

There are a lot of possible choices that can be defended, but this choice made the most sense to me.

I would even propose that we adjust that image you provided by further inspection
(inconsistent coloring with other tabs, semicolon is inconsistent as well, size is an `h2` but all other tabs use an `h1`).

**Comment by stsewd** [comment]
Yeah, you are right! Do you want to open a new issue for the other suggested changes?

**Comment by ze** [comment]
I'll open one right now, thanks for reviewing my pull request!

File: readthedocs/templates/projects/project_dashboard_base.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h3>{% trans "Projects" %}</h3>
     <div class="module search search-dashboard">
       <form action="{% url 'search' %}" method="GET">
         <div class="text-input-wrapper search search-dashboard">
-          <input type="text" name="q" class="search">
```
Comment (xrmx) [inline readthedocs/templates/projects/project_dashboard_base.html]: Please make the string translatable

Comment (ze) [reply readthedocs/templates/projects/project_dashboard_base.html]: Alright, it's been done!

**Comment by ericholscher** [comment]
Thanks!

**Pull request merged**